cms_frontend/src/pages/Dashboard.tsx

```diff
@@ -41,45 +38,146 @@ const Dashboard: React.FC = () => {
 
   // Moves our current directory up (analogous to `cd ..`)
   const toParent = () => {
-    const parent = getParent() as FolderName;
+    const parent = getParent();
     setDir(parent);
+    setFolder(Files.get(parent) as FileFormat[]);
   }
 
   // Checks if our current directory has a parent directory
   const hasParent = () => {

```

Comment:
A bit redundant imo, since all folders has a parent except for root dir, so maybe instead of using a for loop just check that the CWD is root dir?